policy-controller/src/admission.rs

```diff
@@ -44,7 +81,10 @@ pub fn routes(client: kube::Client) -> BoxedFilter<(impl warp::Reply,)> {
                 Ok(servers) => servers,
                 Err(error) => {
                     warn!(%error, "Failed to list servers");
-                    return ok(http::StatusCode::INTERNAL_SERVER_ERROR);
+                    return Response::builder()
+                        .status(http::StatusCode::INTERNAL_SERVER_ERROR)
+                        .body(Body::empty())
+                        .map_err(|e| e.into());
```

Comment:
again, take it or leave it, but this should never fail...

Reply:
an anyhow::Result is expected here so mapping the err seems just as good as unwrapping and rewrapping.

Reply:
I think Eliza's point is that this can't actually fail at runtime. The error is really there to report user errors, so using `.expect("response must be valid")` is fine